homeassistant/components/unifiprotect/strings.json

```diff
@@ -61,24 +61,24 @@
     }
   },
   "issues": {
-    "ea_warning": {
-      "title": "UniFi Protect v{version} is an Early Access version",
+    "ea_channel_warning": {
+      "title": "UniFi Protect Early Access enabled",
       "fix_flow": {
         "step": {
           "start": {
-            "title": "v{version} is an Early Access version",
-            "description": "You are using v{version} of UniFi Protect which is an Early Access version. [Early Access versions are not supported by Home Assistant](https://www.home-assistant.io/integrations/unifiprotect#about-unifi-early-access) and it is recommended to go back to a stable release as soon as possible.\n\nBy submitting this form you have either [downgraded UniFi Protect](https://www.home-assistant.io/integrations/unifiprotect#downgrading-unifi-protect) or you agree to run an unsupported version of UniFi Protect."
+            "title": "UniFi Protect Early Access enabled",
+            "description": "You are either running an Early Access version of UniFi Protect (v{version}) or you have opt-ed into a release channel that is not the Official stable release channel. [Early Access versions are not supported by Home Assistant](https://www.home-assistant.io/integrations/unifiprotect#about-unifi-early-access) and you should switch back to the Official stable release channel as soon as possible. Accidentally upgrading to an Early Access version can break your UniFi Protect integration.\n\nBy submitting this form you have switched back to the official release channel or you agree to run an unsupported version of UniFi Protect which may break your Home Assistant integration at anytime."
```

Comment:
```suggestion
            "description": "You are either running an Early Access version of UniFi Protect (v{version}) or you have opt-ed into a release channel that is not the official stable release channel. [Early Access versions are not supported by Home Assistant](https://www.home-assistant.io/integrations/unifiprotect#about-unifi-early-access) and you should switch back to the official stable release channel as soon as possible. Accidentally upgrading to an Early Access version can break your UniFi Protect integration.\n\nBy submitting this form you have switched back to the official release channel or you agree to run an unsupported version of UniFi Protect which may break your Home Assistant integration at anytime."
```

Reply:
Those were captialized to make it stand out and match the name that appears in the UI for Protect.

![image](https://github.com/home-assistant/core/assets/490848/38e15e4a-95f1-4c69-b4c3-418f1a5b0b81)